TestPlaybooks/NonCircleTests/playbook-BMCHelixRemedyforce-Test.yml

```diff
@@ -0,0 +1,1645 @@
+id: BMCHelixRemedyforce-Test
```

Comment:
@Itay4 @liorblob Why did we add this test to the `NoneCircleTests` folder?

Reply:
because we don't have an instance.